x-pack/legacy/plugins/siem/public/components/event_details/event_details.test.tsx

```diff
@@ -15,6 +15,9 @@ import { EventDetails } from './event_details';
 import { mockBrowserFields } from '../../containers/source/mock';
 import { defaultHeaders } from '../../mock/header';
 
+jest.mock('../../lib/settings/use_kibana_ui_setting');
```

Comment:
As any FYI to anyone who stumbles on this: if a test you're developing fails with `Error: Uncaught [TypeError: Cannot read property 'uiSettings' of null]`, it's because the `.test.tsx` is missing this line.